packages/app-store/basecamp3/components/EventTypeAppCardInterface.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState, useEffect } from "react";
+
+import { useAppContextWithSchema } from "@calcom/app-store/EventTypeAppContext";
+import AppCard from "@calcom/app-store/_components/AppCard";
+import type { EventTypeAppCardComponent } from "@calcom/app-store/types";
+import { Select } from "@calcom/ui";
+
+import type { appDataSchema } from "../zod";
+
+const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({ app }) {
+  const [getAppData, setAppData] = useAppContextWithSchema<typeof appDataSchema>();
+  const [enabled, setEnabled] = useState(getAppData("enabled"));
+  const [projects, setProjects] = useState();
+  const [selectedProject, setSelectedProject] = useState<undefined | { label: string; value: string }>();
+
+  useEffect(() => {
+    fetch("/api/integrations/basecamp3/projects")
```

Comment:
Having my doubts about using `fetch` here for an internal API, not sure if it is expected to have apps to do this, and if so, maybe there is an standard way to do it. Summoning @hariombalhara JIC

Reply:
This is a first and unique case in this respect so far. I would prefer to use tRPC only here. Any challenges in using that @Jonathan1599

Reply:
I'm not sure. I haven't used tRPC before. Is there an app integration that's currently using it? I could use it as a reference

Reply:
Check this out https://github.com/calcom/cal.com/blob/1504715a0458f8d12e4f738ecb9eace9647287f3/packages/app-store/routing-forms/components/FormActions.tsx#L14

Reply:
Thanks!